prSHARK/backends/github.py

```diff
@@ -480,92 +552,69 @@ def parse_review_comment(self, mongo_pr, mongo_prr, prrc):
-        Parses and stores a commit related to a pull request in the database.
+        if 'in_reply_to_id' in prrc:
+            new_prrc.in_reply_to_id = str(prrc['in_reply_to_id'])
 
+        if (self.pr_id, self.review_id) not in self.parsed_prs['review_comments']:
+            self.parsed_prs['review_comments'][(self.pr_id, self.review_id)] = {}
+        self.parsed_prs['review_comments'][(self.pr_id, self.review_id)][str(prrc['id'])] = new_prrc
+        self.get_diff(mongo_prrc, new_prrc, 'pull_request_review_id')
 
-        :param mongo_pr: The associated pull request in the MongoDB.
-        :param pr_commit: A dictionary containing commit information.
-        :return: None.
+    def get_diff(self, old, new, ex_path):
         """
-        try:
-            mongo_pr_commit = PullRequestCommit.objects.get(pull_request_id=mongo_pr.id, commit_sha=pr_commit['sha'])
-        except PullRequestCommit.DoesNotExist:
-            mongo_pr_commit = PullRequestCommit(pull_request_id=mongo_pr.id, commit_sha=pr_commit['sha'])
-        # author and commiter urls are not always present, sometimes we only have a username and an email but no login
-
-        if pr_commit['author'] and 'url' in pr_commit['author'].keys():
-            mongo_pr_commit.author_id = self._get_person(pr_commit['author']['url'])
-        else:
-            mongo_pr_commit.author_id = self._get_person_without_url(pr_commit['author']['name'],
-                                                                     pr_commit['author']['email'])
+        Compare and identify differences between old and new objects.
 
-        if pr_commit['committer'] and 'url' in pr_commit['committer'].keys():
-            mongo_pr_commit.committer_id = self._get_person(pr_commit['committer']['url'])
-        else:
+        This function compares two objects, `old` and `new`, typically representing data from different
+        states, to identify differences between them. The comparison is performed by utilizing the
+        DeepDiff library. Differences are stored in the `pr_diff` dictionary for the pull request
+        identified by `self.pr_id`. If differences are found, the corresponding key in `pr_diff`
+        is set to `True`.
 
-            mongo_pr_commit.committer_id = self._get_person_without_url(pr_commit['committer']['name'],
-                                                                       pr_commit['committer']['email'])
+        :param old: The old object to be compared.
+        :param new: The new object to be compared.
+        :param ex_path: List of paths to be excluded from the comparison.
 
-        mongo_pr_commit.parents = [p['sha'] for p in pr_commit['parents']]
-        mongo_pr_commit.message = pr_commit['message']
-        mongo_pr_commit.commit_repo_url = self._get_repo_url(pr_commit['url'])
-        mongo_pr_commit.commit_id = self._get_commit_id(mongo_pr_commit.commit_sha, mongo_pr_commit.commit_repo_url)
-        mongo_pr_commit.save()
+        """
```

Comment:
Just two remarks. The name implies the method returns something. You can also set the self.pr_diff[self.pr_id] = False at the beginning. That would indicate to the reader that this is the only function that sets this marker. Otherwise the reader would have to look where else pr_diff[self.pr_id] is set and which other values it could take.